js/util/keyMap.js

```diff
@@ -19,7 +19,8 @@ var defaultKeyMap = {
   'closeAllTabs': 'shift+mod+n',
   'reload': 'mod+r',
   'showAndHideMenuBar': 'ctrl+m',
-  'followLink': 'mod+enter'
+  'followLink': 'mod+enter',
+  'passwordAutofillShortcut': 'mod+a'
```

Comment:
This is already a built-in shortcut on Mac (it's "select all"), so we'll need to pick a different key combination for this.

Reply:
Any suggestions? I'm fine with whatever key is free :)

Reply:
1password uses cmd/ctrl + backslash, would that work?